src/components/AutoCompleteTags/AutoCompleteTags.story.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+
+import withTests from '../../util/withTests';
+import AutoCompleteTags from './AutoCompleteTags';
+
+const randomItems = [];
+const mails = ['@sumup.com', '@gmail.com', '@hotmail.com'];
+const names = ['liam', 'josh', 'tom', 'adam', 'aaron', 'john', 'killjoy'];
+const pushRandom = () => {
```

Comment:
Maybe we could think of a seeding library for tests? Something like faker maybe.

Reply:
We should consider this. I've used `chance` in the past, but don't have any preference.